libs/libimobiledevice/Makefile

```diff
@@ -41,7 +41,7 @@ define Package/libimobiledevice
   $(call Package/libimobiledevice/Default)
   SECTION:=libs
   CATEGORY:=Libraries
-  DEPENDS:=+libplist +libimobiledevice-glue +libusbmuxd +libopenssl
+  DEPENDS:=+libplist +libimobiledevice-glue +libtatsu +libusbmuxd +libopenssl
   LICENSE:=LGPL-2.1-or-later
   LICENSE_FILES:=COPYING.LESSER
 endef
```

Comment:
In row 58, there is typo. Missing L in "LICENSE_FILES".

Reply:
Fixed typo: `ICENSE_FILES` -> `LICENSE_FILES`